tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -465,51 +419,54 @@ describe('IssueList', function () {
           data: expect.stringContaining(''),
         })
       );
-
-      expect(getSearchBarValue(wrapper)).toBe('is:resolved');
-
-      // Organization saved search selector should have default saved search selected
-      expect(getSavedSearchTitle(wrapper)).toBe('My Pinned Search');
     });
 
-    it('selects a saved search and changes sort', async function () {
+    it('selects a saved search', async function () {
       const localSavedSearch = {...savedSearch, projectId: null};
       savedSearchesRequest = MockApiClient.addMockResponse({
         url: '/organizations/org-slug/searches/',
         body: [localSavedSearch],
       });
       createWrapper();
-      await tick();
-      await tick();
-      wrapper.update();
 
-      wrapper.find('SavedSearchTab DropdownMenu a').simulate('click');
-      wrapper.find('SavedSearchMenuItem a').last().simulate('click');
+      await waitFor(() => {
+        expect(wrapper.queryByTestId('loading-indicator')).toBe(null);
```

Comment:
Why `findBy` vs `queryBy`?

Reply:
because `findBy querie`s can be used to await the changes in the DOM

Reply:
https://testing-library.com/docs/dom-testing-library/api-async#findby-queries

Reply:
Ahh that makes sense

Reply:
So actually, I think we have to use `waitFor` here because we're waiting for the element to *not* exist, I don't think this can be done with `findBy` queries.